readme.md

```diff
@@ -2,21 +2,23 @@
 
 ## Learning Objectives
 
-* Create a new rails application with Postgres as the default database.
-* Use `rails` commands to create, edit, and update, and seed the db.
-* Use Rails generators to create migrations.
-* Use Rails console to inspect and manipulate models.
-* Use Rails migrations to create tables and modify columns.
-* Undo a migration with `rails db:rollback`.
-* Create migrations that associate one model with another.
-* Identify the impacts of editing existing migrations.
-* Use `timestamps` to timestamp crud actions.
-* Use shorthand syntax to create migrations from the command line.
+* Create a new rails application with Postgres as the default database
+* Use `rails` commands to create, edit, update and seed a database
+* Use Rails generators to create migrations
+* Use Rails console to inspect and manipulate models
+* Use Rails migrations to create tables and modify columns
+* Undo a migration with `rails db:rollback`
+* Create migrations that associate one model with another
+* Identify the impacts of editing existing migrations
+* Use `timestamps` to timestamp CRUD actions
+* Use shorthand syntax to create migrations from the command line
 
 ---
 
 ## Framing
 
+<!-- AM: Could use some more framing... -->
+
```

Comment:
Yea, could talk about how we saw a framework for the first time with Sinatra, and again briefly explain why we use frameworks. So there's the line i said (andy kim invented) this round about rails being the nuclear power fire hose or whatever, and how we started with sinatra as the garden hose. gotta learn the garden hose before you can use nuclear powered one. oh look, now you get the nuclear powered hose. I think the key thing to highlight here is that we're still dealing with a similar idea as sinatra, but it's just gonna look a little different. really, connect them with what they already know